telemetry-http-okhttp/src/main/java/com/newrelic/telemetry/OkHttpPoster.java

```diff
@@ -25,12 +25,20 @@ public OkHttpPoster() {
     this(Duration.ofSeconds(2));
   }
 
-  /** Create an OkHttpPoster with a default OkHttpClient, with only a custom call timeout. */
+  /**
+   * Create an OkHttpPoster with a default OkHttpClient, with only a custom http timeout.
+   *
+   * @param callTimeout expected connection timeout
```

Comment:
Let's change this from `connection timeout` to `call timeout`.  The javadocs for OkHttp explain that "* The call timeout spans the entire call: resolving DNS, connecting, writing the request body, server processing, and reading the response body.", so it's more than just connect.